src/modules/LocalStorage.js

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable linebreak-style */
```

Comment:
- [ ] As i have suggested above, you are not meant to use any linters diable rule in your codebase. The linters are there to make sure your code meets the best standard practices. I'll suggest you remove this linter diable rule and fix the issue. :+1: :+1: :+1:

Reply:
Hello, @V-Blaze, thank you for reviewing my code. I have an issue with the eslint error "Expected line breaks to be LF but found CRLF". I have tried fixing even in my previous projects  but it is still recurring on all JS FILES. Could you suggest any changes on that.

Reply:
Thank you I finally managed to solve the eslint error.

Reply:
Fixed.